src/lib/viewers/iframe/IFrameLoader.js

```diff
@@ -33,6 +33,12 @@ class IFrameLoader extends AssetLoader {
         // The IFrame viewer is disabled when the file is a Boxdicom file and the disableDicom viewer option is enabled
         if (disableDicom && isDicomFile) {
             disabledViewers.push('IFrame');
```

Comment:
Do we still need this logic now that the extension is removed?

Reply:
Yeah I don't think we need it anymore, will remove.